.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: ci
```

Comment:
```suggestion
name: tests
```

Reply:
![cititles](https://user-images.githubusercontent.com/5493666/192085967-c4a8b37f-a607-4235-8633-29bb1b1a0319.png)

The final result. Might be better to have something like "ci / tests" and other jobs like "ci / packaging check" what do you think ?